recipes/crashpad/all/conanfile.py

```diff
@@ -190,47 +200,48 @@ def build(self):
             "extra_cflags_cc=\\\"{}\\\"".format(" ".join(extra_cflags_cc)),
             "extra_ldflags=\\\"{}\\\"".format(" ".join(extra_ldflags)),
         ]
-        with tools.chdir(self._source_subfolder):
-            with self._build_context():
-                self.run("gn gen out/Default --args=\"{}\"".format(" ".join(gn_args)), run_environment=True)
-                targets = ["client", "minidump", "crashpad_handler", "snapshot"]
-                if self.settings.os == "Windows":
-                    targets.append("crashpad_handler_com")
-                self.run("ninja -C out/Default {targets} -j{parallel}".format(
-                    targets=" ".join(targets),
-                    parallel=tools.cpu_count()), run_environment=True)
+        with chdir(self, self.source_folder):
+            parsed_gn_args = " ".join(gn_args)
+            self.run(f'gn gen out/Default --args="{parsed_gn_args}" --script-executable="{sys.executable}"', env="conanbuild")
```

Comment:
In this instance I would refrain from referencing `sys.executable` in recipes. Bear in mind that `conan` can be (and actually is) distributed as self-contained bundles where the interpreter is embedded in the executable - I'd imagine that this recipe might not work if the python interpreter (as seen by Conan) is not a standalone interpreter.

If the gn scripts referenced `python`, I would guess this is because they supported and at one point expected Python 2.7. 
I can see in the crashpad repository that they now reference python3: https://github.com/chromium/crashpad/commit/f88a116c0e2e6f38c348a889fb28921510955802

I'd imagine that `--script-executable=python3` might suffice if the python scripts themselves are also compatible with python3 which they appear they are. It would fallback on finding the `python3` in the system or the currently activated environment, but this wouldn't be any different than expecting `gcc`, `cmake` or `ninja` to be available.